src/modality-specific-files/physiological-recordings.md

```diff
@@ -66,118 +164,639 @@ Additional metadata may be included as in
+}
+```
+
+The decompressed contents of the corresponding TSV file,
+`sub-01_task-nback_physioevents.tsv.gz` could read now:
+
+```tsv
+13894432325	Ready
+13894432331	Synchronous recalibration triggered
+13894432334	External message received: new block
+```
+
+## Specific physiological signal types
+
+### Eye-tracking
+
+<!--
+1. Datasets will be updated later to adapt to the agreed format.
+2. We aim at adding a last example converting published dataset from openeuro.
+-->
+!!! example "Example datasets"
+
+    [Example datasets](https://bids-standard.github.io/bids-examples/#dataset-index)
+    with eye-tracking data have been formatted using this specification
+    and can be used for practical guidance when curating a new dataset:
+
+    -   Combined fMRI and eye-tracking data in a resting-state task, measured with
+        an Eyelink (SR research). Human participant kept their gaze steady at the
+        screen center.
+
+        [BIDS dataset](https://openneuro.org/datasets/ds004158/versions/2.0.1)
+
+    -   Combined behavioral and eye-tracking data, measured with and Eyelink
+        (SR Research). Human participants freely viewed as set of natural images.
+        Published paper: https://doi.org/10.1523/ENEURO.0287-23.2023
+
+        [BIDS dataset](https://github.com/julia-pfarr/natImSac_BIDSexample)
+
+Setting `PhysioType` to the keyword `"eyetrack"` specifies that
+the physiological recordings in the `<matches>_physio.tsv.gz` have
+been acquired with an *eye-tracker*.
+In the following, *eye-tracker* refers to the apparatus allowing
+the recording of gaze position, and, optionally, pupil size.
+
+Eye-tracking data MUST be stored following the general specifications
+for `"generic"` physiological recordings.
+However, it is REQUIRED that recordings corresponding to each eye
+(and/or *cyclopean* or averaged signals for binocular eye-trackers providing
+a third recording) are split into files with different
+[`recording-<label>`](../appendices/entities.md#recording).
+Therefore, the use of [`recording-<label>`](../appendices/entities.md#recording)
+is REQUIRED with eye-tracking data.
+The values `"eye1"`, `"eye2"`, and `"eye3"` are RECOMMENDED as the respective labels
+for the [`recording-<label>`](../appendices/entities.md#recording) entity.
+
+!!! danger "MANDATORY metadata"
+
+    The correspondence of labels and the recorded eye MUST be encoded
+    by the MANDATORY `RecordedEye` metadata.
+
+    The [`recording-<label>`](../appendices/entities.md#recording) entity
+    MAY take other values such as `"left"`, `"cyclopean"`, or `"right"` corresponding
+    to the `RecordedEye` metadata.
+    However, it is RECOMMENDED that metadata is not encoded in the file names to avert
+    that filename and metadata enter into conflict.
+    For example, if [`recording-<label>`](../appendices/entities.md#recording) takes
+    the value `"left"` but the corresponding sidecar JSON file contains a definition of
+    `RecordedEye` being `"right"`.
```

Comment:
If you're making a RECOMMENDATION, then presumably the validator should warn if the value is not `eye1`, `eye2` or `eye3`. You would need to write a check.

I wonder if a cleaner proposal would be:

```suggestion
    To avoid filenames and metadata coming into conflict,
    if the `recording` label starts with `left` or `right`,
    then the `RecordedEye` field MUST be `"left"` or `"right"`, respectively.
    For example `recording-left` or `recording-lefteye` MUST
    match to `"RecordedEye": "left"` in the corresponding JSON sidecar file.
```

I just suspect that you're going to get a lot of people wanting to use `left` and `right` in the filename, and erroring on mismatch will be more effective than warning on use.

Reply:
This is the same problem of dir-. You can have `dir-AP` and `PhaseEncodingDirection` = `i` with a RAS affine and the validator can't do much about it.

What the validator can impose is that eye tracking physio recordings must have a label with pattern `/eye[1-9]`.

would that be okay?

Reply:
> This is the same problem of dir-. You can have `dir-AP` and `PhaseEncodingDirection` = `i` with a RAS affine and the validator can't do much about it.

We can check this, and I'm up for making it happen.

Keeping `right`/`left` correct is also *much easier* than `AP/PA`.

> What the validator can impose is that eye tracking physio recordings must have a label with pattern `/eye[1-9]`.

Yes, this can be done. In either case, you need MUST language and an error, not a recommendation.

I'll defer to BEP020 leads to decide which will be more acceptable to users.